docs/standard/serverless-architecture/architecture-approaches/architecture-deployment-approaches.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Serverless apps. Architecture, patterns, and Azure implementation.
+description: Serverless apps. Architecture, patterns, and Azure implementation. | Architecture deployment approaches
+keywords: Serverless, Microservices, .NET, Azure, Azure Functions
+author: JEREMYLIKNESS
+ms.author: jeliknes
+ms.date: 1/26/2018
+ms.prod: .net
+ms.technology: dotnet
+ms.topic: article
+---
+# Architecture deployment approaches
+
+Regardless of the architecture approach used to design a business application, the implementation, or deployment of those applications may vary. Businesses host applications on everything from physical hardware to serverless functions.
+
+## N-Tier applications
+
+The [N-Tier architecture pattern](/azure/architecture/guide/architecture-styles/n-tier) is a mature architecture and simply refers to applications that separate various logical layers into separate physical tiers. N-Tier architecture is a physical implementation of N-Layer architecture. The most common implementation of this architecture includes:
+
+* A presentation tier, for example a web app
+* An API and/or data access tier, such as a REST API
+* A data tier, such as a SQL database
+
+![N-tier architecture](./media/n-tier-architecture.png)
```

Comment:
the art files should be under ./media/<topic name>/artfile.jpg

for more info see https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-art?branch=master#naming-and-placement-of-media-files